test/hotspot/jtreg/runtime/cds/appcds/TestParallelGCWithCDS.java

```diff
@@ -116,8 +116,10 @@ static void test(boolean dumpWithParallel, boolean execWithParallel, boolean use
                 } else {
                     String pattern = "((Too small maximum heap)" +
                                        "|(GC triggered before VM initialization completed)" +
-                                       "|(java.lang.OutOfMemoryError: Java heap space))";
-                    out.shouldMatch(pattern);
+                                       "|(java.lang.OutOfMemoryError: Java heap space)" +
+                                       "|(Initial heap size set to a larger value than the maximum heap size))";
+                        out.shouldMatch(pattern);
+                        out.shouldNotHaveFatalError();
```

Comment:
The indentation is off here. All the double-quotes should line up and the `out.shouldMatch` line should be shifted left two spaces. Thanks

Reply:
Fixed.